products/memgraph/v0.51.0/reference_guide/query-modules.md

```diff
@@ -6,33 +6,8 @@ startup.
 
```

Comment:
Above:
`Memgraph supports extending the query language with user written procedures.` ?
`These procedures are grouped into modules, which can then be loaded on startup.` ?